assets/js/src/marketing_optin_block/edit.tsx

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable react/react-in-jsx-scope */
+/* eslint-disable import/no-unresolved */
+
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useBlockProps, RichText } from '@wordpress/block-editor';
+import { CheckboxControl } from '@woocommerce/blocks-checkout';
+import { getSetting } from '@woocommerce/settings';
+import { Placeholder, Button } from '@wordpress/components';
+import { Icon, megaphone } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import './editor.scss';
+
+const adminUrl = getSetting('adminUrl') as string;
+const { optinEnabled, defaultText } = getSetting('mailpoet_data');
+
+const EmptyState = (): JSX.Element => (
+  <Placeholder
+    icon={<Icon icon={megaphone} />}
+    label={__('Marketing opt-in', 'mailpoet')}
+    className="wp-block-mailpoet-newsletter-block-placeholder"
+  >
+    <span className="wp-block-mailpoet-newsletter-block-placeholder__description">
+      {__(
+        'MailPoet marketing opt-in would be shown here if enabled. You can enable from the settings page.',
```

Comment:
I am afraid this is not going to work. We don't use the standard WordPress translation function. You can check other places where is used `MailPoet.I18n.t`.